test/vscoutDart_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:test/test.dart';
+import 'package:sembast/sembast.dart';
+import 'package:sembast/sembast_io.dart';
+import 'package:path/path.dart';
+import 'dart:io';
+
+String dbPath = 'test.db';
+
+Future main() async {
+  Database db = await databaseFactoryIo
+      .openDatabase(join(".dart_tool", "sembast", "example", dbPath));
+  Store store = db.getStore("main");
+  Record record = Record(store, {"name": "ugly"});
+  record = await db.putRecord(record);
+  record = await store.getRecord(record.key);
+  record =
+      (await store.findRecords(Finder(filter: Filter.byKey(record.key)))).first;
+  print(record);
+  record = await store.getRecord(record.key);
+
+  var records =
+      (await store.findRecords(Finder(filter: Filter.matches("name", "^ugl"))));
+  print(records);
+}
+// void main() {
```

Comment:
I"ll just assume ur not done with ur test

Reply:
Lol